src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -56,18 +60,28 @@ const CanonicalMostRead = ({
 
   useEffect(() => {
     if (!items) {
-      const handleResponse = async response => {
+      const handleResponse = url => async response => {
         if (!response.ok) {
-          throw Error(response.statusText);
+          throw Error(
+            `Unexpected response (HTTP status code ${response.status}) when requesting ${url}`,
+          );
         }
         const mostReadData = await response.json();
         setItems(processMostRead({ data: mostReadData, numberOfItems }));
       };
 
-      const fetchMostReadData = pathname =>
-        fetch(pathname, { mode: 'no-cors' })
-          .then(handleResponse)
-          .catch(e => logger.error(`HTTP Error: "${e}"`));
+      const fetchMostReadData = pathname => {
+        logger.info(MOST_READ_CLIENT_REQUEST, { url: endpoint });
+
+        return fetch(pathname, { mode: 'no-cors' })
+          .then(handleResponse(pathname))
+          .catch(error => {
+            logger.error(MOST_READ_FETCH_ERROR, {
+              url: pathname,
+              message: error.toString(),
```

Comment:
Is it worth renaming `message` to `error`?  In the logs, the event & message are already output, e.g. 
```
2020-05-20 08:49:08.512 error [filename] {
  "event": "event_name_here",
  "message": {
     "url": "/path/",
    "error": "error string here",
...
    }
```

Reply:
I've updated this in my most recent commit